diff --git a/dist/index.mjs b/dist/index.mjs
index 5b75602818c90215aca94704a0756918653e4b6b..361c2e07b70734099b5bf716356ff3742901ee19 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -412,6 +412,8 @@ var generateOperationId = (method, paths) => {
 };
 var cloneHook = (hook) => {
   if (!hook) return;
+  if (typeof hook === 'string') return hook
+	if (Array.isArray(hook)) return [...hook]
   return { ...hook };
 };
 var registerSchemaPath = ({
